src/main/java/com/github/sh0nk/matplotlib4j/TypeConversion.java

```diff
@@ -17,4 +18,8 @@
             }
         }).collect(Collectors.toList());
     }
+
+    public String toSafeDouble(Double d) {
```

Comment:
Can this arg be changed to unboxed `double`, since if `null` is passed accidentally, that is stringified as `null` which is invalid for python?

Reply:
I've just added a commit.